doc/release-notes/4259-java-11.md

```diff
@@ -0,0 +1,91 @@
+
+## Notes to Admins
+
+### Java 11
+
+There are some things to note and keep in mind regarding the move to Java 11:
+
+- You should install the JDK/JRE following your usual methods, depending on your operating system.
+- We targeted and tested Java 11, but 11+ will likely work. Java 11 was targeted because of its long term support.
+- You will need to adjust some JVM options in domain.xml as part of the upgrade process. We've provided these steps below.
+
+### New JVM and DB Options
+
+#### New JVM Options
+
+The following JVM Options have been added in support of Java 11:
+
+<jvm-options>[9|]--add-opens=java.base/jdk.internal.loader=ALL-UNNAMED</jvm-options>
+<jvm-options>[9|]--add-opens=jdk.management/com.sun.management.internal=ALL-UNNAMED</jvm-options>
+<jvm-options>[9|]--add-exports=java.base/jdk.internal.ref=ALL-UNNAMED</jvm-options>
+<jvm-options>[9|]--add-opens=java.base/java.lang=ALL-UNNAMED</jvm-options>
+<jvm-options>[9|]--add-opens=java.base/java.net=ALL-UNNAMED</jvm-options>
+<jvm-options>[9|]--add-opens=java.base/java.nio=ALL-UNNAMED</jvm-options>
+<jvm-options>[9|]--add-opens=java.base/java.util=ALL-UNNAMED</jvm-options>
+<jvm-options>[9|]--add-opens=java.base/sun.nio.ch=ALL-UNNAMED</jvm-options>
+<jvm-options>[9|]--add-opens=java.management/sun.management=ALL-UNNAMED</jvm-options>
+<jvm-options>[9|]--add-opens=java.base/sun.net.www.protocol.jrt=ALL-UNNAMED</jvm-options>
+<jvm-options>[9|]--add-opens=java.base/sun.net.www.protocol.jar=ALL-UNNAMED</jvm-options>
+<jvm-options>[9|]--add-opens=java.naming/javax.naming.spi=ALL-UNNAMED</jvm-options>
+<jvm-options>[9|]--add-opens=java.rmi/sun.rmi.transport=ALL-UNNAMED</jvm-options>
+<jvm-options>[9|]--add-opens=java.logging/java.util.logging=ALL-UNNAMED</jvm-options>
+
+## Upgrade Instructions
+
+0\. These instructions assume that you've already successfully upgraded from Dataverse 4.x to  Dataverse 5 following the instructions in the [Dataverse 5 Release Notes](https://github.com/IQSS/dataverse/releases/tag/v5.0).
+
+1\. Undeploy the previous version.
+
+- `<payara install path>/bin/asadmin list-applications`
+- `<payara install path>/bin/asadmin undeploy dataverse<-version>`
+
+(where `<payara install path>` is where Payara 5 is installed, for example: `/usr/local/payara5`)
+
+2\. Stop Payara and remove the generated directory
+
+- `service payara stop`
+- remove the generated directory: 
+`rm -rf <payara install path>/payara/domains/domain1/generated`
+
+3\. In domain.xml:
+
+Remove the following JVM options:
+
+  <jvm-options>-Djava.endorsed.dirs=/usr/local/payara5/glassfish/modules/endorsed:/usr/local/payara5/glassfish/lib/endorsed</jvm-options>
+  459a448
+  <jvm-options>-Djava.ext.dirs=${com.sun.aas.javaRoot}/lib/ext${path.separator}${com.sun.aas.javaRoot}/jre/lib/ext${path.separator}${com.sun.aas.instanceRoot}/lib/ext</jvm-options>
+
+Add the following JVM options to the domain><configs><config name-"server-config"><java-config> element:  
```

Comment:
This path is also where you need to delete the two options above. (It shouldn't hurt if you delete them in both places they show up, but it might be worth just moving ~this line up.

Reply:
updated in https://github.com/IQSS/dataverse/pull/7514/commits/9a2802f400dc21b47d25c948ada66e9c3366336c